backend/src/utils/resourceUtils.ts

```diff
@@ -592,6 +592,10 @@ export const getDashboardConfig = (): DashboardConfig => {
   return dashboardConfigWatcher.getResources()?.[0];
 };
 
+/**
+ * Feature flags are required in the config -- but upgrades can be mixed and omission of the property
+ * usually ends up being enabled. This will prevent that as a general utility.
+ */
 export const featureFlagEnabled = (disabledSettingState?: boolean): boolean =>
```

Comment:
This function is going away in #2010 -- what is the OSSM need here? You just work or not work based on the feature flag? Do you have any backend component stack (in the DSC) that we can look at?